config

```diff
@@ -15,6 +15,9 @@ DRYRUN="false"
 VERBOSE="false"
 EXE=""
 THERE=`dirname $0`
+# How long to wait after a warning, in seconds
+SLEEPTIME=50
+#SLEEPTIME=0
```

Comment:
Come on, man! 50 seconds? Note that in `stty -icanon min 0 time 50` interval is measured in 10th of second, so that 50 is 5 seconds.

Reply:
Let me open an request instead...